includes/class-api.php

```diff
@@ -349,6 +349,9 @@ public function attach_featured_image( $post_id, $post_args ) {
 			return false;
 		}
 
+		// Allow download_url() to use an external host to retrieve featured image.
+		add_filter( 'http_request_host_is_external', array( $this, 'allow_sync_external_host' ), 10, 3 );
```

Comment:
We need to return true to this filter to allow an external request to be made when we call [download_url()](https://github.com/themarcusbattle/press-sync/blob/master/includes/class-api.php#L221) this filter defaults to false and this causes an error to be returned when we attempt to download the featured image.

Filter Info:
https://developer.wordpress.org/reference/hooks/http_request_host_is_external/